Tempo/Controllers/PlayerTableViewController.swift

```diff
@@ -95,6 +105,21 @@ class PlayerTableViewController: UITableViewController, UISearchResultsUpdating,
 		removeRevealGesture()
 	}
 	
+	// Filter posted dates into dictionary of key: date, value: date_count
```

Comment:
Is there a reason for why this method is in `PlayerTableViewController` as opposed to `PostHistoryTableViewController`? Do the Feed and Liked view controllers need it?